lib/peripheral.dart

```diff
@@ -82,12 +130,22 @@ class Peripheral {
         transactionId ?? TransactionIdGenerator.getNextId(),
       );
 
+  /// Fetch list of Descriptors
```

Comment:
Should use third-person form.
```suggestion
  /// Fetches a list of [Descriptor]s.
```